components/brave_shields/resources/panel/container.tsx

```diff
@@ -36,7 +36,7 @@ function Container () {
       return (<TreeList
         data={siteBlockInfo?.jsList}
         totalBlockedCount={siteBlockInfo?.jsList.length}
-        blockedCountTitle={getLocale('braveShieldsScriptsBlocked')}
```

Comment:
Let's delete from grd for `braveShieldsScriptsBlocked` if it's not used anymore.